install.md

```diff
@@ -50,6 +52,8 @@ Then to install Buildah on Fedora follow the steps in this example:
   buildah --help
 ```
 
+## RHEL 7
```

Comment:
Probably should add CENTOS 7 here as well.

Reply:
alright, fixed up to say "RHEL, CentOS" instead. i haven't changed the specific instructions, assuming they are the same in CentOS...